AUTHORS

```diff
@@ -0,0 +1,7 @@
+Ariel Gabizon
```

Comment:
please include first all existing authors from the AUTHORS file that we derive from (requirement of MIT license)